### DISCUSSION TITLE
Pagination rendering extended-minified selection.

### DISCUSSION THREAD

**Discussion by andresin87** [SUI-Components/sui-components#1527] [Ideas and new features]
When js is disabled in desktop front-side (google-bots), the ssr render pages. Some of them have the pagination component. The minimum with of it exceeds the width of the page and makes the browser display it with an undesired (higher) viewport (scroll Y).

[MTR-40932](https://jira.scmspain.com/browse/MTR-40932)

Can we offer the minimized mobile version for this cases?

@AgonisticKatai

**Comment by turolopezsanabria** [comment]
hello @andresin87 @AgonisticKatai  still valid?
